torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9069,9 +9069,6 @@ def ref_pairwise_distance(input1, input2):
                     assert_autodiffed=True,
                     rhs_make_tensor_kwargs=dict(exclude_zero=True),
                     skips=(
-                        # 69913: RuntimeError: CUDA error: an illegal memory access was encountered
-                        DecorateInfo(unittest.skip("Skipped!"), 'TestGradients', 'test_fn_fwgrad_bwgrad',
-                                     device_type='cuda', dtypes=[torch.double, torch.cdouble]),
                         DecorateInfo(unittest.skip("Skipped!"), 'TestGradients', 'test_forward_mode_AD',
```

Comment:
After patching this PR,  plain forward mode AD tests also pass. We should remove those skips as well.

Reply:
Could you also double check for the inplace ones?

Reply:
I'll submit a follow-up PR to remove more skips. First let's get this fix in after rocm test passes